### PR TITLE
Update ttlafterfinished.md

### PR DESCRIPTION
**Pull request by my-git9**
The ttlafterfinshed doc is updated with English documentation

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 1]
```diff
@@ -1,65 +1,54 @@
 ---
-title: 已完成资源的 TTL 控制器
+title: 已完成作业的自动清理
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md:2]: Job 不翻译

File: content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 2]
```diff
@@ -1,65 +1,54 @@
...
-such as Pods and custom resources.
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} provides a 
+TTL (time to live) mechanism to limit the lifetime of resource objects that 
+have finished execution. TTL controller only handles 
+{{< glossary_tooltip text="Jobs" term_id="job" >}}.
 -->
-TTL 控制器提供了一种 TTL 机制来限制已完成执行的资源对象的生命周期。
-TTL 控制器目前只处理 {{< glossary_tooltip text="Job" term_id="job" >}}，
-可能以后会扩展以处理将完成执行的其他资源，例如 Pod 和自定义资源。
+TTL-after-finished {{<glossary_tooltip text="controller" term_id="controller">}} 提供了一种 TTL 机制来限制已完成执行的资源对象的生命周期。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md:22]:
```suggestion
TTL-after-finished {{<glossary_tooltip text="控制器" term_id="controller">}} 提供了一种
TTL 机制来限制已完成执行的资源对象的生命周期。
```

File: content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md (hunk 1 above)
Comment (my-git9) [reply content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md]: Made some adjustments as suggested，thanks for your review

File: content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md (hunk 2 above)
Comment (my-git9) [reply content/zh/docs/concepts/workloads/controllers/ttlafterfinished.md]: Made some adjustments as suggested，thanks for your review

**Pull request closed without merging**